### PR TITLE
Remove *hybrid images from 4.7 and beyond

### PR DESCRIPTION
**Pull request by jmrodri**

### CODE REVIEW

File: ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-master.yaml [hunk 1]
```diff
@@ -99,6 +90,14 @@ tests:
     from: src
 - as: sanity
   commands: |
+    echo "Debug statements"
+    pwd
+    id
+    echo "setting environment"
+    export GO111MODULE=on
+    export GOFLAGS=-mod=vendor
```
Comment (joelanford) [inline ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-master.yaml:98]: We may have already talked about this, but ...

Do we need to explicitly set `-mod=vendor`? I feel like I remember that being the default in openshift/release go builder images, and only having to override it if we wanted something else.

**Review by joelanford — APPROVED** [review]
/hold
for one nit.

otherwise
/lgtm

File: ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-master.yaml (hunk 1 above)
Comment (jmrodri) [reply ci-operator/config/openshift/ocp-release-operator-sdk/openshift-ocp-release-operator-sdk-master.yaml]: Removed both of these.

**Pull request merged**